packages/esm-patient-chart-app/src/visit/visit-form/visit-form.component.tsx

```diff
@@ -106,6 +112,57 @@ const StartVisitForm: React.FC<DefaultWorkspaceProps> = ({ patientUuid, closeWor
         .subscribe(
           (response) => {
             if (response.status === 201) {
+              if (config.showServiceQueueFields) {
+                const status = statuses.find((data) => data.display.toLowerCase() === 'waiting').uuid;
+                const defaultPriority = priorities.find((data) => data.display.toLowerCase() === 'not urgent').uuid;
```

Comment:
```suggestion
                const status = statuses.find((data) => data.display.toLowerCase() === 'waiting').uuid;
                const defaultPriority = priorities.find((data) => data.display.toLowerCase() === 'not urgent').uuid;
```

Reckon these default values should be driven by config as well?